actions/xpubs/get.go

```diff
@@ -47,3 +47,42 @@ func (a *Action) get(c *gin.Context) {
 	contract := mappings.MapToXpubContract(xPub)
 	c.JSON(http.StatusOK, contract)
 }
+
+// get will get an existing model
+// Get xPub godoc
+// @Summary		Get xPub
+// @Description	Get xPub
+// @Tags		xPub
+// @Produce		json
+// @Success		200 {object} models.Xpub "xPub associated with the given xPub from auth header"
+// @Failure		500	"Internal Server Error - Error while fetching xPub"
+// @Router		/v1/users/current [get]
+// @Security	x-auth-xpub
+func (a *Action) get2(c *gin.Context) {
```

Comment:
I know that it's pretty big change to you're work, but you should place all the endpoints from this PR in a separate package, because they are in scope of "user" not "xpub" and it will be hard to search for them in the future.

In other words, you need to create new package similar to xpubs package and put there those 2 endpoints and   whole setup needed to make them working.

Reply:
Moved to package `users`